### PR TITLE
feat(COR-888): add labels support

### PR DESCRIPTION
**Pull request by pggb25**

### CODE REVIEW

File: internal/domain/labels_group/labels_group.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+)
+
+var (
+	ErrInvalidLabelsGroupRequest             = errors.New("invalid aabels group request")
+	ErrInvalidLabelsGroupOrganizationIdParam = errors.New("invalid organization id format")
+	ErrFailedToCreateLabelsGroup             = errors.New("failed to create aabels group")
+	ErrFailedToGetLabelsGroup                = errors.New("failed to get aabels group")
+	ErrFailedToUpdateLabelsGroup             = errors.New("failed to update aabels group")
+	ErrFailedToDeleteLabelsGroup             = errors.New("failed to delete aabels group")
+	ErrInvalidLabelsGroupIdParam             = errors.New("invalid aabels group id format")
```
Comment (mzottola) [inline internal/domain/labels_group/labels_group.go:16]: typo:
```suggestion
	ErrInvalidLabelsGroupRequest             = errors.New("invalid labels group request")
	ErrInvalidLabelsGroupOrganizationIdParam = errors.New("invalid organization id format")
	ErrFailedToCreateLabelsGroup             = errors.New("failed to create labels group")
	ErrFailedToGetLabelsGroup                = errors.New("failed to get labels group")
	ErrFailedToUpdateLabelsGroup             = errors.New("failed to update labels group")
	ErrFailedToDeleteLabelsGroup             = errors.New("failed to delete labels group")
	ErrInvalidLabelsGroupIdParam             = errors.New("invalid labels group id format")
```

File: docs/data-sources/application.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ data "qovery_application" "my_application" {
 - `environment_variable_overrides` (Attributes Set) List of environment variable overrides linked to this application. (see [below for nested schema](#nestedatt--environment_variable_overrides))
 - `environment_variables` (Attributes Set) List of environment variables linked to this application. (see [below for nested schema](#nestedatt--environment_variables))
 - `healthchecks` (Attributes) Configuration for the healthchecks that are going to be executed against your service (see [below for nested schema](#nestedatt--healthchecks))
+- `labels_group_ids` (Set of String) list of labels group ids
```
Comment (mzottola) [inline docs/data-sources/application.md:42]: To be consistent with existing doc (same for others `.md` files)
```suggestion
- `labels_group_ids` (Set of String) List of labels group ids
```

**Review by mzottola — APPROVED** [review]

File: internal/domain/labels_group/labels_group.go (hunk 1 above)
Comment (pggb25) [reply internal/domain/labels_group/labels_group.go]: fixed. Thank you

File: docs/data-sources/application.md (hunk 2 above)
Comment (pggb25) [reply docs/data-sources/application.md]: fixed. Thank you

**Pull request merged**